packages/azure-kusto-data/src/connectionBuilder.ts

```diff
@@ -3,6 +3,7 @@
 
 import { DeviceCodeInfo, InteractiveBrowserCredentialInBrowserOptions, InteractiveBrowserCredentialNodeOptions } from "@azure/identity";
 import { KustoConnectionStringBuilderBase } from "./connectionBuilderBase";
+import fs from "fs";
```

Comment:
This might fail in the browser - check

Reply:
There is a connectionBuilder.browser.ts file